src/proxy.c

```diff
@@ -1383,10 +1384,7 @@ cc_proxy_run_hyper_new_container (struct cc_oci_config *config,
 	json_object_set_string_member (process, "workdir",
 			config->oci.process.cwd);
 
-	/* Hyperstart is not able to handle uid 0. This is a bug.
-	 * Simply dont sent the uid for now as the container
-	 * runs as root by default until the bug is resolved.
-	 */
+	// Hyperstart is not able to handle uid 0. This is a bug.
```

Comment:
Sure, I guessed as much, but the comment already existed. The point is, this is no longer the minimal change. 99% of the codebase uses C-style comments even for single-line comments, so new code should strive to honour the existing conventions... even if it is "only a comment". There are only 2 other C++ style comments in that file, and they relate to `FIXME`'s which by definition require that code (comment) to be rewritten anyway.

Aside: Ideally, this comment would also reference the hyperstart bug to help the reader, which might bump it into a multi-liner anyway.